mesheryctl/internal/cli/root/connections/delete.go

```diff
@@ -0,0 +1,87 @@
+package connections
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+
+var deleteConnectionCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a connection",
+	Long: `Delete
+a connection`,
+
+	Example: `
+// Delete a connection
+mesheryctl system connections delete <connection_id>
```

Comment:
```suggestion
mesheryctl system connections delete [connection_id]
```

Reply:
Solved.

Reply:
I don;t see the suggestion applied.

Reply:
@RipulHandoo ?

Reply:
Unresolving conversation until this is confirmed, @RipulHandoo

Reply:
This is outdated.